temp.c

```diff
@@ -173,65 +259,22 @@ void temp_sensor_tick() {
 
 				#ifdef	TEMP_THERMISTOR
 				case TT_THERMISTOR:
-					do {
-						uint8_t j, table_num;
-						//Read current temperature
-						temp = analog_read(i);
```

Comment:
I haven't read this thread till now. The reason why I need separate ADC is that I have 5 temperature sensors in a print bed and 4 heaters. Print bed is 40x40cm with four 20x20 heaters. One sensor sits in the middle of each heater and the fifth on is in the middle of a test bed. You may see it in the video I tweeted some time ago: https://twitter.com/WitekSowa/status/713456028860473344
This number of sensors and heaters plus LCD, jog dial, SD and at least 5 endstops (max in Z axis, min/max in X/Y) requires much more pins than ATMega have so I run ADC/SD via SPI and LCD/Jog dial via I2C port multiplexer (MCP23017).